content/en/build-guide/fan-box.adoc

```diff
@@ -0,0 +1,286 @@
+---
+title: "How to build a PAPRa Fan Box"
+linkTitle: "PAPRa Fan Box"
+weight: 1
+description: >
+  How-To guide for building the Tetra PAPRa fan box (March 2021 version).
+---
+
+== Quick Synopsis ==
+
+The Tetra Powered Air Purified Respirator requires that the air be purified; this purification is accomplished by powering a fan to pull air through a filter and then pushing that purified air through a hose to a mask worn on the face.  This section contains the instructions for building the box that holds the fan and filter.
+
+Please note that this version of the fanbox has a known problem-- the power connector specified on the PCB is proprietary, and does not properly link into the provided DC cables.  The next version of the fan box will fix this problem.
```

Comment:
I'd also convert this text block to an [admonition](https://docs.asciidoctor.org/asciidoc/latest/blocks/admonitions/), prepended with the **NOTE** keyword